openjdk/playlist.xml

```diff
@@ -125,6 +115,38 @@
 		<groups>
 			<group>openjdk</group>
 		</groups>
+		<impls>
+			<impl>hotspot</impl>
+		</impls>
+	</test>
+	<test>
+		<testCaseName>hotspot_custom_j9</testCaseName>
+		<variations>
+			<variation>Mode150</variation>
+			<variation>Mode650</variation>
+			<variation>Mode1000</variation>
+		</variations>
+		<command>$(JAVA_COMMAND) -Xmx512m -jar $(Q)$(TEST_RESROOT)$(D)jtreg$(D)lib$(D)jtreg.jar$(Q) \
+	$(JTREG_BASIC_OPTIONS) -nativepath:"$(TESTIMAGE_PATH)/hotspot/jtreg/native" -vmoptions:$(Q)-Xmx512m $(JVM_OPTIONS) $(VMOPTION_HEADLESS)$(Q) \
```

Comment:
This only works for jvmti tests. Would be better to change the testCaseName.

Reply:
Yes, we only support serviceability tests for now. But this provides the ability to test/run other tests if needed. As the name suggested `hotspot_custom_j9`, users have the option to choose what tests to run.

Reply:
For other tests which nativepath `-nativepath:"$(TESTIMAGE_PATH)/hotspot/jtreg/native"` or `-nativepath:"$(TESTIMAGE_PATH)$(D)openj9" ` work? I thought for openj9 serviceability tests is different as it need hotspot lib while others still need openj9 lib. I might be wrong, just to confirm.

Reply:
I can see that `[hotspot_native_sanity](https://github.com/adoptium/aqa-tests/blob/master/openjdk/playlist.xml#L424)` and `[ASGCTBaseTest_j9](https://github.com/adoptium/aqa-tests/blob/master/openjdk/playlist.xml#L389)` are using openj9 lib path nativepath:"$(TESTIMAGE_PATH)$(D)openj9". That is this new openj9 hotspot custom target might not work for those tests. I'm suggesting to restore the hotspot_custom as is and add this new jvmti custom for openj9.

Reply:
This test is to offer openj9 the capability to test with hotspot native path, the `JvmtiGetSystemPropertyTest` is only an example to show it works, but we can also specify other hotspot native tests as the custom_target when needed.
  The `hotspot_native_sanity` and `ASGCTBaseTest_j9` is to test j9 native path, instead of the hotspot native path as this test. They serve different purposes.
  The `hotspot_custom` test is already restored for impl hotspot only in this PR. Thanks.

Reply:
My suggestion is restore hotspot_custom for hotspot and openj9 as before, so openj9 hotspot_native_sanity and ASGCTBaseTest_j9 can also use the hotspot_custom as nativepath has been correspondingly set to j9 native path.

This new hotspot_custom_j9 won't work with openj9 hotspot_native_sanity and ASGCTBaseTest_j9

Reply:
The hotspot_custom can be restored for both hotspot and openj9, but it still needs to be disabled for j9, since no default custom target will pass all versions and platforms so far ( `hotspot_native_sanity` is disabled now, and `ASGCTBaseTest_j9` is xlinux and 16+ only). Shall we do it this way?

Reply:
yes, in that way hotspot_custom can be disabled for j9.

Reply:
For the naming issue with `hotspot_custom_j9` and `nativepath -nativepath:"$(TESTIMAGE_PATH)/hotspot/jtreg/native"`  ,  my question is if it works for hotspot_native_sanity and ASGCTBaseTest_j9 (those are also hotspot tests)? My answer is no.  I don't think this provide the ability to run other openjdk hotspot tests for openj9 if openj9 use openj9 native path. Are you going to add another target for openj9 to run hotspot tests using openj9 native libs?

Reply:
I do not understand why we are so focused on hotspot_native_sanity and ASGCTBaseTest_j9? As Longyu mentioned, hotspot_custom_j9 is designed for tests that hotspot native lib, not for tests that use openj9 native lib. That is, it is not designed for hotspot_native_sanity and ASGCTBaseTest_j9.

btw, hotspot_native_sanity has not been enabled for openj9 since the beginning. ASGCTBaseTest_j9 runs a single java file, so it does not need to use custom target to run.

Reply:
Updated the commit to restore `hotspot_custom` as suggested. Thanks.